### ISSUE TITLE
What does <data path> mean? 

### ISSUE THREAD

**Issue by KyosukeSenda** [turningpoint1988/FCNA#1]
Excuse me.
We have now prepared all requirements in README.
Also, we copied hg19.fa into directory "/project/FCNA/hg19/" whose "/FCNA" was a directory we cloned this repository.
Then, we have a question in data pre-processing.
Does `<data path>` mean "/project/FCNA/hg19/"?
We tried but had an error 
`FileNotFoundError: [Errno 2] No such file or directory: '/home/**/FCNMotif/hg19/hg19.fa'`
Would you give us some idea?

**Comment by turningpoint1988** [comment]
这是来自QQ邮箱的自动回复邮件。您好，您所发的邮件已收到，我会尽快给您回复。

**Comment by turningpoint1988** [comment]
> Hello, it means a specified directory which stores all TF datasets, such as 'A549' or 'GM12878' in this repository.

**Comment by KyosukeSenda** [comment]
I appreciate your rapid response.
Now, we did `bash process.sh A549` in  `/project/FCNA`.
Then, we frequently have an error 
`FileNotFoundError: [Errno 2] No such file or directory: '/home/**/FCNMotif/hg19/hg19.fa'`
Do you assume that we prepare dir. `/hg19` & a file hg19.fa in dir. `/FCNMotif`?

I feel very sorry for asking again and again.

**Comment by turningpoint1988** [comment]
> I appreciate your rapid response. Now, we did `bash process.sh A549` in `/project/FCNA`. Then, we frequently have an error `FileNotFoundError: [Errno 2] No such file or directory: '/home/**/FCNMotif/hg19/hg19.fa'` Do you assume that we prepare dir. `/hg19` & a file hg19.fa in dir. `/FCNMotif`?
> 
> I feel very sorry for asking again and again.

Of course, you can create a dir. named hg19 in this repository and then put the downloaded hg19.fa into it.  
I have updated the script 'chip_processing.py', and pls use it.

**Comment by KyosukeSenda** [comment]
I appereciate your help and we broke through.
By the way, in run.sh, I think we could not make a directory with `mkdir ./models/~/`.
We should do `mkdir -p ./models/~/` instead.

**Comment by turningpoint1988** [comment]
> I appereciate your help and we broke through. By the way, in run.sh, I think we could not make a directory with `mkdir ./models/~/`. We should do `mkdir -p ./models/~/` instead.

Thanks for your advices. I will update it as soon as possible.
